core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -0,0 +1,346 @@
+// you may not use this file except in compliance with the License.
```

Comment:
I guess you missed the starting three lines of this comment?

Reply:
Did not noticed that.